### ISSUE TITLE
Screen Stacking Animation Issue

### ISSUE THREAD

**Issue by darrylsyms** [darrylsyms/fretwise-app#8]
In some areas of the app there seems to be a navigation stacking issue. One screen closes whilst the other opens, instead of stacking. Here are three examples with videos. You might need to slow the videos down as the transition happens quick, but if you look closely you can see exactly what is happening. Where possible, this has been tested in both the custom and default environments, and is true in both settings.

**1. Topics Screen**
- This is within the Topics List Screen. Press a Topic Item, then "Reply", then if you cancel the reply it takes you right back to the Topics List Screen.

https://user-images.githubusercontent.com/82924984/152471172-8dfe0d70-b9fc-429a-99d3-75d31b716d28.MP4

**2. Profile Screen**
- When a list item is clicked inside the `ProfileScreen`, the stack navigation seems to crossover. When you’re inside a list item screen and you press ‘back’, it takes you back to the moreScreen. Expected behaviour is to go back into the ProfileScreen.

https://user-images.githubusercontent.com/82924984/152470842-3f986e03-9d9c-44d3-a4ca-9e2881587ff6.MP4

**3. Lesson Screen**
- This is within a lesson. When an image is pressed, it opens in a new stack, but that simultaneously closes the lessonSingleScreen, and so when the user goes "back" they are taken to the courseSingleScreen...

https://user-images.githubusercontent.com/82924984/152300506-07e51002-fc3c-4f3f-86a1-c4751d07a3e2.mov

**Comment by darrylsyms** [comment]
This isn't uncommon throughout the app. The most problematic area that this occurs is within lessons/topics. When an image is pressed, it opens in a new stack, but that simultaneously closes the lessonSingleScreen, and so when the user goes "back" they are taken to the courseSingleScreen...

https://user-images.githubusercontent.com/82924984/152300506-07e51002-fc3c-4f3f-86a1-c4751d07a3e2.mov

**Comment by darrylsyms** [comment]
This bug can be reproduced by using the `navigationApi.setScreensWithoutTabBar` hook! Utilise this code:

```
    externalCodeSetup.navigationApi.setScreensWithoutTabBar([
        "LessonSingleScreen",
        "LearnTopicSingleScreen",
        "TopicsSingleScreen",
        "AddTopicScreen",
        "MessagesCreatePostScreen",
        "GroupMessageCreatePostScreen",
        "BlogSingleScreen",
        "ProfileScreen"
    ])
```

**Comment by darrylsyms** [comment]
Fixed in 1.5.x
